apps/namadillo/src/App/Governance/AllProposalsTable.tsx

```diff
@@ -3,14 +3,10 @@ import { useState } from "react";
 import { GoCheckCircleFill, GoInfo } from "react-icons/go";
 import { useNavigate } from "react-router-dom";
 
-import {
-  Stack,
-  StyledSelectBox,
-  StyledTable,
-  TableRow,
-} from "@namada/components";
+import { Stack, StyledSelectBox, TableRow } from "@namada/components";
 import { Proposal, isProposalStatus, proposalStatuses } from "@namada/types";
 import { Search } from "App/Common/Search";
+import { TableWithPaginator } from "App/Common/TableWithPaginator";
```

Comment:
Just a question: Is this now generic enough to live in `@namada/components`? I don't think it's important to move it soon, if at all, just a thought!